components/Form.svelte

```diff
@@ -5,14 +5,29 @@
     TextArea,
     TextInput,
   } from "carbon-components-svelte";
-  import { gjScheme, clearCurrentlyEditing } from "../stores.js";
+  import {
+    gjScheme,
+    clearCurrentlyEditing,
+    currentlyEditing,
+  } from "../stores.js";
 
   export let id;
   export let name;
   export let intervention_type;
   export let description;
   export let length_meters;
 
+  let bottomOfForm;
+
+  currentlyEditing.subscribe((openedId) => {
+    if (id == openedId) {
+      // Wait 1ms before doing this, because it appears the accordion doesn't
```

Comment:
I can't figure out how to listen to events for when the accordion finishes expanding